Makefile

```diff
@@ -407,15 +407,16 @@ endif
 	sed -i.bak 's/$(PREVIOUS_VERSION)/$(RELEASE_CANDIDATE)/g' ./cmd/otelcorecol/builder-config.yaml
 	sed -i.bak 's/$(PREVIOUS_VERSION)/$(RELEASE_CANDIDATE)/g' examples/k8s/otel-config.yaml
 	find . -name "*.bak" -type f -delete
-	# regenerate files
-	$(MAKE) -C cmd/builder config
-	$(MAKE) genotelcorecol
 	# commit changes before running multimod
 	git checkout -b opentelemetry-collector-bot/release-$(RELEASE_CANDIDATE)
 	git add .
 	git commit -m "prepare release $(RELEASE_CANDIDATE)"
 	$(MAKE) multimod-prerelease
-	# commit multimod changes
+	# merge multimod changes
+	git merge prerelease_collector-core_v$(RELEASE_CANDIDATE)
+	# regenerate files
+	$(MAKE) -C cmd/builder config
+	$(MAKE) genotelcorecol
```

Comment:
need to delete the branch?

Reply:
Also, what about https://github.com/open-telemetry/opentelemetry-go-build-tools/pull/134 ?

Reply:
having multimod not require a new branch would be great. i'll review that change tomorrow 👍🏻

Reply:
Ping?

Reply:
PR was merged, consider to bump version and use that.

Reply:
Updated to use latest multimod w/ `-b=false` set to prevent it from creating a separate branch